packages/gasket-plugin-docusaurus/lib/index.js

```diff
@@ -5,6 +5,42 @@ module.exports = {
   name: require('../package').name,
   hooks: {
     configure,
-    docsView
+    docsView,
+    metadata(gasket, meta) {
+      return {
+        ...meta,
+        configurations: [{
+          name: 'docusaurus',
+          link: 'README.md#configuration',
+          description: 'Docusaurus plugin config',
+          type: 'object',
+          default: '{}'
```

Comment:
I'm not sure having `default: '{}'` for these top-level config objects is helpful. I'd say let's just leave defaults off for these.
```suggestion
```